api-tester/src/main/java/com/revenuecat/apitester/kotlin/GoogleProrationModeAPI.kt

```diff
@@ -7,7 +7,10 @@ private class GoogleProrationModeAPI {
     fun check(mode: GoogleProrationMode) {
         when (mode) {
             GoogleProrationMode.IMMEDIATE_WITHOUT_PRORATION,
-            GoogleProrationMode.IMMEDIATE_WITH_TIME_PRORATION -> {}
+            GoogleProrationMode.IMMEDIATE_WITH_TIME_PRORATION,
+            GoogleProrationMode.DEFERRED,
+            GoogleProrationMode.IMMEDIATE_AND_CHARGE_FULL_PRICE,
+            GoogleProrationMode.IMMEDIATE_AND_CHARGE_PRORATED_PRICE -> {}
```

Comment:
We need to add these to the java version of these API tests as well

Reply:
Done!